modal/modal.html

```diff
@@ -9,142 +9,142 @@
 <body>
 <div id="share-directory">
     <div id="share-directory-grid" class="clearfix">
-        <button class="col share clearfix" id="facebook" data-share="https://www.facebook.com/sharer.php?u">
+        <button class="col share clearfix" id="facebook" title=“Facebook” data-share="https://www.facebook.com/sharer.php?u">
```

Comment:
Button tag doesn't have any `title` parameter, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button doesn't need that.

Reply:
I'd add it as `alt` attribute to the `<img />` within?

Reply:
Can get more sense to img tag instead of button, but is something that can be useful?

Reply:
Of course!
Right now, the `<button>`s wouldn't be announced in a meaningful way to screenreaders.